doc/api/tls.md

```diff
@@ -880,8 +880,8 @@ publicly trusted list of CAs as given in
   * `honorCipherOrder` {boolean} When choosing a cipher, use the server's
     preferences instead of the client preferences. Defaults to `true`.
   * `requestCert` {boolean} If `true` the server will request a certificate from
-    clients that connect and attempt to verify that certificate. Defaults to
-    `false`.
+    clients that connect and attempt to verify that certificate against
+    certificates provided in `ca` option. Defaults to `false`.
```

Comment:
Okay, then I totally misunderstood that option. If they are to be replaced, it certainly makes sense to provide a cert bundle instead.